### PR TITLE
Robby dermody patch 1

### PR DESCRIPTION
**Pull request by hansieodendaal**
To close PR #58

### CODE REVIEW

**Comment by hansieodendaal** [comment]
Robby's changes merged into branch

**Comment by hansieodendaal** [comment]
Just wait, need to update the presentations quickly

**Comment by hansieodendaal** [comment]
All done, please review @CjS77 

Note: See updated Counterparty S/W stack drawing in sources and report, will be referenced correctly from the PITCHME.md in master

File: layer2scaling/more-landscape/PITCHME.md [hunk 1]
```diff
@@ -69,11 +69,12 @@ Has benefits to Tari as a trustless Masternode matching/batch processing engine
 
 ---
 
-#### #2 Counterparty Eco System
+#### #2 Counterparty
 
-- Counterparty NOT a block chain -> protocol & network of nodes for smart contract apps, using EVM linked to Bitcoin block chain
-- Embedded consensus (identical ledgers, no P2P network) & federated nodes
-- Tx meta data are written into Bitcoin Txs on Bitcoin block chain
+- Counterparty NOT a block chain -> token protocol that operates on Bitcoin
```
Comment (CjS77) [inline layer2scaling/more-landscape/PITCHME.md]: Please use full(er) sentences

File: layer2scaling/more-landscape/PITCHME.md [hunk 2]
```diff
@@ -69,11 +69,12 @@ Has benefits to Tari as a trustless Masternode matching/batch processing engine
...
+- Full DEX & hardcoded smart contracts, e.g. difference, binary options
```
Comment (CjS77) [inline layer2scaling/more-landscape/PITCHME.md]: Decentralised Exchange (DEX)

File: layer2scaling/more-landscape/PITCHME.md [hunk 3]
```diff
@@ -102,13 +104,14 @@ Has benefits to Tari as a trustless Masternode matching/batch processing engine
 
 <u>Strengths</u>
 
-- Provides smart contract abilities rooted in Bitcoin block chain
-- Embedded consensus - nodes maintain identical ledgers without P2P network
-
+- Provides hard coded smart contract abilities rooted in Bitcoin block chain
+- Embedded consensus - nodes maintain identical ledgers without P2P network. "permission less innovation"
```
Comment (CjS77) [inline layer2scaling/more-landscape/PITCHME.md]: permissionless

File: layer2scaling/more-landscape/landscape-update.md [hunk 4]
```diff
@@ -64,36 +64,46 @@ Has benefits to Tari as a trustless Masternode matching/batch processing engine
...
+
+In the past, projects like [Storj](https://storj.io/) and [SWARM](https://counterparty.io/news/swarm-crowdfunding/) also built on Counterparty. 
 
 COVAL is being developed with a primary purpose of moving value using “off-chain” methods. They use their own set of node runners to manage various "off-chain" distributed ledgers and ledger assigned wallets to implement an extended transaction value system, whereby tokens as well as containers of tokens can be securely transacted. Scaling within the COVAL eco system is thus achievable because they are not only reliant on the Counterparty federated nodes to execute smart contracts. [[33]](https://coval.readme.io/docs)
 
 #### Strengths
 
-- Counterparty provides smart contract abilities that is rooted in the Bitcoin block chain
+- Embedded consensus/Counterparty provides a simple way to add "layer 2" functionality to an already existing blockchain implementation that supports basic data embedding
+- Counterparty's embedded consensus model utilizes "permission less innovation", meaning that even the Bitcoin core developers could not stop the use of the protocol layer without seriously crippling the network.
```
Comment (CjS77) [inline layer2scaling/more-landscape/landscape-update.md]: permissionless

File: layer2scaling/more-landscape/PITCHME.md (hunk 3 above)
Comment (hansieodendaal) [reply layer2scaling/more-landscape/PITCHME.md]: Done, in both the report and presentation

File: layer2scaling/more-landscape/landscape-update.md (hunk 4 above)
Comment (hansieodendaal) [reply layer2scaling/more-landscape/landscape-update.md]: Done

File: layer2scaling/more-landscape/PITCHME.md (hunk 1 above)
Comment (hansieodendaal) [reply layer2scaling/more-landscape/PITCHME.md]: Done

File: layer2scaling/more-landscape/PITCHME.md (hunk 2 above)
Comment (hansieodendaal) [reply layer2scaling/more-landscape/PITCHME.md]: Definition of term added in the report, but not in the presentation

**Comment by hansieodendaal** [comment]
Thank you @robby-dermody for your contribution, I really appreciate your effort, all merged into master now.
Closing #58

**Pull request merged**